flags/mod.ts

```diff
@@ -785,3 +785,120 @@ export function parse<
 
   return argv as Args<TArgs, TDoubleDash>;
 }
+
+/**
+ * Calculates the Levenshtein distance between two strings.
+ *
+ * @param {string} str1 - The first string.
```

Comment:
Types in `@param` directive seem redundant as they are given in TypeScript. We usually omit them in our docs (For example, see https://github.com/denoland/deno_std/blob/6b1da3240ca5f9460df2571096d64566a7cb3c38/async/pool.ts#L31